localization/xliff/enu/constants/localizedConstants.enu.xlf

```diff
@@ -125,6 +125,9 @@
         <trans-unit id="azureAddAccount">
             <source xml:lang="en">Add an Account...</source>
         </trans-unit>
+        <trans-unit id="cannotConnect">
+            <source xml:lang="en">Cannot connect due to expired tokens</source>
```

Comment:
Maybe something like `Cannot connect due to expired tokens. Please reauthenticate and try again.`? This just seems a bit abrupt.